docs/add-scalardb-to-your-build.md

```diff
@@ -1,20 +1,37 @@
-# Add ScalarDB to your build
+# Add ScalarDB to Your Build
```

Comment:
I wonder, should the title be something like the following instead?

```suggestion
# Add the ScalarDB Build to Your Application
```

Reply:
We normally call such a library "dependency". So, IMO, "Add dependency on ScalarDB to your build configuration"  would be more precise, but it is verbose...

Reply:
Can we consider ScalarDB as a build as it is a finished product, not an unfinished one?
Do you think the original title is too vague?

Reply:
@Torch3333 Thanks for that link! I realize now that my previous suggestion was incorrect.

I feel the original title is a little vague, but maybe just adding `Application` helps clarify it. What do you think?

```suggestion
# Add ScalarDB to Your Application Build
```

Reply:
I think it's fine as is since another major project also says the same?
(or, we took it from the guava project maybe.)
https://github.com/google/guava#adding-guava-to-your-build

Reply:
Alright. I'm OK with it as is.